CMakeLists.txt

```diff
@@ -1,83 +1,81 @@
-cmake_minimum_required(VERSION 3.9)
-add_definitions(-DOS_MACOSX)
-project(echo)
+cmake_minimum_required(VERSION 3.2)
```

Comment:
Why change this?

Reply:
We were using Ubuntu 16.04 in our CI systems, and it doesn't have CMake 3.9.
Since ```set(CMAKE_CXX_STANDARD 11)``` was introduced in CMake 3.1, and as we aren't using any advanced features of CMake, I set the CMake minimum version to 3.2.

I can confirm that everything works just as fine on my cmake version 3.14.1 too though/